Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -76,6 +79,10 @@
     <Word from="deo" to="dio" />
     <Word from="Deo" to="Dio" />
     <Word from="dešava" to="događa" />
+    <Word from="dete" to="dijete" />
+    <Word from="djete" to="dijete" />
+    <Word from="Dete" to="Dijete" />
+    <Word from="Djete" to="Dijete" />
```

Comment:
Line 82 and 83 can be merged.

<Word from="([dD])j?ete" to="$1ijete" />

Ps: Ignore https://github.com/SubtitleEdit/subtitleedit/pull/1959#pullrequestreview-487002